### ISSUE TITLE
@extend causing a coredump

### ISSUE THREAD

**Issue by sveetch** [dahlia/libsass-python#165]
Hi,

I found an obscure bug that cause a coredump, here you can see a sample source on SassMeister:

https://gist.github.com/sveetch/99fa86f9716b020d1cb19e869b20971a

Using it from commandline :

`sassc sample.scss`

It results in:

`Segmentation fault (core dumped)`

And nothing else in the output.

Basically i was trying to compile this using my Boussole project, so through your `sass` module and finally tried directly with sassc commandline tool to isolate the bug.

Finally i tried this sample code on SassMeister with any available compiler and always have a correct return output. Also i successfully compiled this code with Compass 1.0.1.

**Comment by asottile** [comment]
This is likely fixed by [3.3.6](https://github.com/sass/libsass/releases/tag/3.3.6) See sass/libsass#2017 sass/libsass#2031 which we cannot upgrade to due to https://github.com/sass/libsass/issues/2046 (We tried to here: https://github.com/dahlia/libsass-python/pull/149).

We'll probably have to wait until they drop 3.4 (hopefully soon!): https://github.com/sass/libsass/issues/2174

I can confirm your bug:

```
$ sassc --version
sassc 0.11.1 (sass/libsass 3.3.5)
$ sassc test.scss 
Segmentation fault (core dumped)
```

I can also confirm that upgrading (when we can!) will fix it:

```
$ sassc --version
sassc 0.11.1 (sass/libsass 3.3.6-144-g50a6)
$ sassc test.scss
/*
 * Sample for proof of bug.
 *
 * It has been minimalized from original sources to find out the real stuff causing bug.
 */
.marked, .clean-block > .content h2.marked {
  text-decoration: underline; }

@media only screen and (min-width: lower-bound(64.063em, 90em)) {
  .clean-block > .content h2:not(.marked):not(.clean-block > .content h2.marked) {
    padding: 0; } }

@media only screen and (min-width: lower-bound(90.063em, 120em)) {
  .clean-block > .content h2:not(.marked):not(.clean-block > .content h2.marked) {
    font-size: 1rem; } }

```

Going to close this as it's an exact duplicate of an upstream bug and hopefully we'll be able to upgrade soon :)

**Comment by sveetch** [comment]
Ah, i presumed something like this when readed another closed issues here. I'll keep watching about next update.

Thanks for your reply!

**Comment by asottile** [comment]
3.3.6 is now available via  0.11.2 :tada:
